charts/osm/Chart.yaml

```
@@ -18,7 +18,7 @@ version: 1.0.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: latest-main
+appVersion: v1.1.0
```

Also update `version` here too.
```suggestion
appVersion: v1.1.0-rc.1
```